Patch to check for username accounts before creating superuser account

arky opened the pull request:
SImple patch to first check for existing accounts before creating superuser account.  

Testcase: 
1. Create a Facility with the following command: 
$ kolibri manage provisiondevice --facility "Kolibri" --superusername admin --superuserpassword "changeme"  --language_id "en"
2. Re-run the same command again. 
3. You will get a django Auth validation error `django.core.exceptions.ValidationError: [u'An account with that username already exists']"],`

### Summary
This patch adds few lines of code to first check if any account with similar usernames exists before running the create super user command in provisiondevice.py  If an existing account is found, it prints out a warning. 

### Reviewer guidance

This patch effects only the management commands that are run on command line. 


### References
Downstream bug reports: https://github.com/iiab/iiab/issues/1094  https://github.com/iiab/iiab/issues/1387 

----

### Contributor Checklist


PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [x] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Gherkin stories have been updated
- [ ] Unit tests have been updated

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.rst

arky committed:
Check usernames before creating superuser

Check for existing accounts before creating superuser account.

indirectlylit commented:
thanks @arky -

note you have a linting error:

```
kolibri/core/device/management/commands/provisiondevice.py:79:1: E302 expected 2 blank lines, found 1
```

indirectlylit commented:
(FYI installing [pre-commit](https://kolibri-dev.readthedocs.io/en/develop/getting_started.html#pre-commit) can be helpful)

arky commented:
@indirectlylit  Thanks for catching that. I had missed the 'pre-commit' step in my dev environment.

arky committed:
Fix linting error

Attempting to fix linting error 'expected 2 blank lints'.

Refer:
https://github.com/learningequality/kolibri/pull/4762#issuecomment-456233580

'kolibri/core/device/management/commands/provisiondevice.py:79:1: E302
expected 2 blank lines, found 1'

jamalex reviewed (approved):
Awesome, thanks!
